dashboard/app/helpers/aichat_helper.rb

```diff
@@ -0,0 +1,64 @@
+module AichatHelper
+  ASSISTANT = "assistant"
+  USER = "user"
+  BASE_PROMPT = "You are a helpful chatbot for children. Limit your responses to a small paragraph. "
+  INSTRUCTIONS_BEGIN_TOKEN = "[INST]"
+  INSTRUCTIONS_END_TOKEN = "[/INST]"
+  SENTENCE_BEGIN_TOKEN = "<s>"
+  SENTENCE_END_TOKEN = "</s>"
+  MAX_NEW_TOKENS = 300
+  SAGEMAKER_CLIENT = Aws::SageMakerRuntime::Client.new
+  SAGEMAKER_MODEL_ENDPOINT = "mistral-7b-inst-v01" # "BioMistral-7B"
+  TOP_P = 0.9
```

Comment:
@tshaffercodeorg just confirming that this value is okay?

Reply:
Regarding SAGEMAKER_MODEL_ENDPOINT? Our two endpoints currently are `mistral-7b-inst-base` (v0.1) and `mistral-7b-bio` (BioMistral) respectively. The other parameters seem fine to me although I'm unsure what Aws::SageMakerRuntime::Client.new does (probably something with the ruby script)

Reply:
Correction, the above information is for our dev AWS account, not the prod one.

Reply:
Thanks! Confirmed that this is correct with @tshaffercodeorg offline as well.

Reply:
Oh yep, I was specifically talking about the `TOP_P` value - I just set that to some default while testing, but is there another value we should use?